<launch>
    <node pkg="lidar" type="lidar" name="lidar" output="screen">
        <param name="num_iter" value="3" />
        <param name="num_lpr" value="250" />
        <param name="th_seeds" value="0.15" />
        <param name="th_dist" value="0.05" />
        <param name="sensor_height" value="0.3" />

        <param name="clustering_method" value="string" />
        <param name="cluster_tolerance" value="0.4" />
        <param name="point_count_theshold" value="1" />
        <param name="min_distance_factor" value="1.5"/>
        <param name="minimal_curve_intensity" value = "100"/>
        <param name= "minimal_height_cone" value = "0.2"/>
        <param name= "th_floor" value = "0.1"/>
    </node>
</launch>